### ISSUE TITLE
MVC5 support

### ISSUE THREAD

**Issue by neographikal** [SimpleAuthentication/SimpleAuthentication#205]
MVC5 is now in RC status, maybe it's time to add preliminary support for MVC5? :)

**Comment by PureKrome** [comment]
Great question @neographikal !

My first instinctive answer to that is going to be: *NOPE*. From what I've seen and played around with (so far) it _looks_ like ASP.NET Core 1.0 has a _very_ similar conceptual _Authentication_ story, baked right in. It doesn't look like it's tied explicitly to Sql Server or *cringe* Membership/ASP.NET Membership, etc.

For example, [have a quick look at the ASP.NET Core *Security* repo](https://github.com/aspnet/Security/) and [the samples](https://github.com/aspnet/Security/tree/dev/samples) found in that repo, which are doing what SimpleAuthentication was created to do.

In summary: looks like ASP.NET Core 1.0 will replace SimpleAuthentication.

I'm not 100% sure yet, as I've only done some small playing around - but it basically looks like it.
